jobs/migrations/0001_initial.py

```diff
@@ -17,14 +17,17 @@ class Migration(migrations.Migration):
                 ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('title', models.CharField(max_length=100)),
                 ('description', models.CharField(max_length=255)),
-                ('location', models.CharField(max_length=100)),
+                ('location', models.CharField(blank=True,
+                 choices=[('1', 'Tel Aviv'), ('2', 'Jerusalem'), ('3', 'Haifa')],
```

Comment:
so you are supporting just these 3 areas?
how are you going to scale this in the future?

Reply:
Do you suggest we change it to a models.CharField(max_length=100)? instead of an enum

Reply:
That is why we added the blank=True. I can add another option for location as ('4', Other) just in case a job is not in the 3 locations given

Reply:
@yuvalk ?

Reply:
no
again, just trying to suggest you think now how the search will work and see that this model fits that